.github/workflows/coding-standard.yml

```diff
@@ -1,9 +1,52 @@
-name: Coding Standard
+name: M2 Coding Testing
 
 on:
+  push:
+    branches: [ 2.4-develop, develop, phpcs_baseline ]
```

Comment:
I think in the end, this only needs to include `2.4-develop`, right?

Reply:
Yes, for testing purpose I added another branches to review